spring-web/src/test/java/org/springframework/web/client/MessageBodyClientHttpResponseWrapperTests.java

```diff
@@ -0,0 +1,98 @@
+package org.springframework.web.client;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.client.ClientHttpResponse;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ * Unit tests for {MessageBodyClientHttpResponseWrapper}.
+ *
+ * @author Yin-Jui Liao
+ */
+public class MessageBodyClientHttpResponseWrapperTests {
+
+
+	@Test
+	public void testMessageBodyNotExist() throws IOException {
+
+		ClientHttpResponse response = new ClientHttpResponse() {
```

Comment:
Please remove the anonymous inner class and replace it with a Mockito mock.

See `HttpMessageConverterExtractorTests` for an example.

Please do this for both test methods.